### PR TITLE
Added test_default_log_level_of_cli

### PR DESCRIPTION
**Pull request by aujjwal-redhat**
Migrated the test case from glusto-test.
Made modifications as per the functionalities in redant.

Updates : #292
Fixes : #303

Signed-off-by: Ayush Ujjwal <aujjwal@redhat.com>

### CODE REVIEW

**Comment by srijan-sivakumar** [comment]
@nik-redhat one of your test case is being migrated :p

**Comment by aujjwal-redhat** [comment]
> @nik-redhat one of your test case is being migrated :p

Yeah :smiley:

**Comment by srijan-sivakumar** [comment]
> I will add all the volumes and test this once

Just a note that, for now, in migration, lets just migrate with the same set of details..if the glusto-tests says test for dist, lets just test for dist. Because at the end of the day, we'd be comparing glusto-tests with redant after these additions and we wouldn't want to be seeing more than what we were supposed to. Maybe we can enhance the test case later...but for now, I think lets keep it the way it is..

**Comment by aujjwal-redhat** [comment]
Screenshot from 2021-05-11 16-33-53
I had run the framework with DEBUG level and we can see it's showing the same in the logs
Screenshot from 2021-05-11 16-37-48 

Screenshot from 2021-05-11 16-37-15

File: tests/functional/glusterd/test_default_log_level_of_cli.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+from glustolibs.gluster.gluster_base_class import GlusterBaseClass, runs_on
+from glustolibs.gluster.volume_ops import (volume_start, volume_status,
+                                           volume_info, volume_stop)
+
+
+@runs_on([['distributed', 'replicated', 'distributed-replicated',
+           'dispersed', 'distributed-dispersed', 'arbiter',
+           'distributed-arbiter'], ['glusterfs']])
+class TestDefaultLogLevelOfCLI(GlusterBaseClass):
+    def setUp(self):
```
[the remark this answers is not in the document]
Comment (srijan-sivakumar) [reply tests/functional/glusterd/test_default_log_level_of_cli.py]: oh...ok..then no worried, I thought maybe it is just restricted to one particular volume type. Cool then..

File: tests/functional/glusterd/test_default_log_level_of_cli.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        cmd = "ls -tr /tmp/redant/ | tail -1"
+        ret = redant.execute_command(cmd, self.server_list[0])
+        log_file = ret['msg'][0][:-1]
+        cmd = f"cat /tmp/redant/'{log_file}'/main.log | grep -F DEBUG | wc -l"
+        ret = redant.execute_command(cmd, self.server_list[0])
+
+        if int(ret['msg'][0].split("\\")[0]) == 0:
+            redant.logger.info("The default log level is INFO")
+        else:
+            redant.logger.info("The default log level is DEBUG")
```
Comment (nik-redhat) [inline tests/functional/glusterd/test_default_log_level_of_cli.py:62]: The test should fail, if the default is not `INFO`.

Comment (aujjwal-redhat) [reply tests/functional/glusterd/test_default_log_level_of_cli.py:62]: Oh ok. Will raise Exception here then.

File: tests/functional/glusterd/test_default_log_level_of_cli.py [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+        redant.volume_start(self.vol_name, self.server_list[0])
+
+        # Check the default log level of cli.log
+        cmd = 'cat /var/log/glusterfs/cli.log | grep -F "] D [" | wc -l'
+        ret = redant.execute_command(cmd, self.server_list[0])
+
+        if int(ret['msg'][0].split("\\")[0]) == 0:
+            redant.logger.info("The default log level is INFO")
+        else:
+            raise Exception("Default log level not INFO")
```
Comment (nik-redhat) [inline tests/functional/glusterd/test_default_log_level_of_cli.py:59]: What's the difference with using `redant.logger.error`? Will that not end the test with the error message?

Comment (aujjwal-redhat) [reply tests/functional/glusterd/test_default_log_level_of_cli.py:59]: And the issue will be the test will pass if we don't raise exception.

**Comment by srijan-sivakumar** [comment]
@aujjwal-redhat @nishith-vihar , let's also add a link of the test case in glusto-tests repo with the commit message, so that it is easy to review and check if we are properly migrating.

File: tests/functional/glusterd/test_default_log_level_of_cli.py (hunk 3 above)
Comment (nik-redhat) [reply tests/functional/glusterd/test_default_log_level_of_cli.py:59]: Okay cool. So, what will be shown on the CLI?

**Review by nik-redhat — APPROVED** [review]
Good that you started with my TC, now the journey ahead would be easier :stuck_out_tongue_winking_eye:

**Review by srijan-sivakumar — APPROVED** [review]
LGTM

**Pull request merged**